.gitlab-ci.yml

```diff
@@ -13,7 +13,7 @@ build-android-apk:
   stage: build
   only:
     - develop
-    - master
+    - main
```

Comment:
This is not the place, should be on a separated PR

Reply:
sure